.github/workflows/deploy-to-production.yml

```
@@ -39,26 +39,36 @@
         id: build
         with:
           context: .
           file: ./Dockerfile
-          push: true
           platforms: ${{ matrix.platform }}
-          tags: ${{ env.REGISTRY_IMAGE }}:${{ matrix.tag }}
+          outputs: type=image,name=${{ env.REGISTRY_IMAGE }},push-by-digest=true,push=true
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Set formatted platform directory in environment variable 🌿
+        run: echo "PLATFORM_DIR=$(echo "${{ matrix.platform }}" | sed 's/\//-/g')" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:74: Double quote to prevent globbing and word splitting

(shellcheck)